do-dont-ns-selector (#576)
centrica-engineering/nucleus-docs: src/content/docs/components/ns-selector.mdx

## Guidance

:::do

> - Use it to support the label with an `image`.
> - Use it to support the label with a `decoration` (illustration).

Reviewer: Consolidate this.

```suggestion
- Use an `image` or a `decoration` to support the label.
```
